src/status_im/ui/screens/browser/events.cljs

```diff
@@ -39,19 +40,42 @@
  (fn [[message webview]]
    (.sendToBridge webview (types/clj->json message))))
 
+(re-frame/reg-fx
+ :resolve-ens-multihash
+ (fn [{:keys [web3 registry ens-name cb]}]
+   (resolver/content web3 registry ens-name cb)))
+
 (handlers/register-handler-fx
  :browse-link-from-message
  (fn [_ [_ link]]
    {:browse link}))
 
+(handlers/register-handler-fx
+ :ens-multihash-resolved
+ [re-frame/trim-v]
```

Comment:
can you clean up the `trim-v` co-effect, we are removing them to have a more consistent codebase